modeling/main.py

```diff
@@ -0,0 +1,282 @@
+# utility packages
+import os
+from datetime import datetime
+import argparse
+
+import numpy as np
+import matplotlib.pyplot as plt
+timestamp = f'{datetime.now():%Y%m%d-%H%M}'
+
+# machine learning packages
+import wandb
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import pytorch_lightning as pl
+from torch.nn import functional as F
+from torch.utils.data import DataLoader, dataloader, random_split
+import torchvision.transforms as transforms
+
+# dataloaders and segmentation models
+from datasets_sc import SCIZurichDataset
+from unet3d import ModifiedUNet3D
```

Comment:
I didn't try this yet, but it might be better to import directly from `ivadomed` with 

```
from ivadomed.models import ModifiedUNet3D
```

here to increase consistency between ivadomed and this custom script.

Reply:
It's exactly the same as the one in ivadomed (and ms-challenge)! Remember how we were thinking of moving away from having a huge models.py file for ivadomed? I wanted to realize that in-practice and hence thought of having different files for different models. But it is the same, shouldn't make any difference. I can add a comment as well saying that it is the same